code/game/gamemodes/blob/powers.dm

```diff
@@ -123,7 +123,16 @@
 		to_chat(src, "There is another core blob nearby, move more than 15 tiles away from it!")
 		return
 
-	if(!can_buy(100))
+	var/cost = 100
+	var/number_of_cores
+
+	for(var/obj/effect/blob/core/C in world)
```

Comment:
I think the game currently does track the amount of cores in the game. It tells you the amount of cores in the Blob Status tab when you're a blob. I don't know how easy it'd be to work in here, though.